js/src/dropdown.js

```diff
@@ -241,16 +243,28 @@ const Dropdown = (() => {
       return placement
     }
 
+    _getOffset(data) {
+      const popperOffset = $.extend({}, data.offsets, this._config.offset(data.offsets) || {})
+      Object.keys(popperOffset).forEach((key) => {
```

Comment:
Sorry to ask I don't know well `offset` function from Popper.js but why do you need that ?

Reply:
data object has many another keys. We need to change only data.offsets and not other. If we return only extended object with data.offsets - Popper is broken

Reply:
That's right 👍 but instead of a Foreach you just have to do : 
`data.offsets = popperOffset`

Reply:
Yup. Agree :) Sorry don't know how I can stick new changes in this PR https://github.com/romanlex/bootstrap/commit/9be393b3a567e327d07c7513e570b4fb1a3d2d41